golang/cosmos/app/app.go

```diff
@@ -889,7 +889,10 @@ func NewAgoricApp(
 	// Store migrations can only run once, so we use a notion of "primary upgrade
 	// name" to trigger them. Testnets may end up upgrading from one rc to
 	// another, which shouldn't re-run store upgrades.
-	if isPrimaryUpgradeName(upgradeInfo.Name) && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+	if upgradeInfo.Name != "" &&
```

Comment:
Why remove this part of the check? I think it makes it more obvious that nothing should be done if there is no upgrade info recorded. Of course no upgrade info is also not going to be a primary upgrade name, but I prefer the explicit nature of the check as it was before.

Reply:
This change is making it *more* explicit, by not even checking `isPrimaryUpgradeName` when `upgradeInfo.Name` is empty (since, as you note, it cannot be a valid upgrade name and nothing should be done).

Reply:
Oh boy, I was not properly awake when I read the diff. Yeah that should have been like that from the beginning.